frontend/src/pages/project/components/ExportProjectComponents.vue

```diff
@@ -1,44 +1,82 @@
 <template>
-    <div>
+    <div class="space-y-4 my-3">
         <FormRow type="checkbox" v-model="parts.flows">
             Flows
-            <template #description>Export Project Flows</template>
         </FormRow>
-        <FormRow type="checkbox" v-bind:disabled="!justCreds" v-model="parts.creds">
-            Credentials
-            <template #description>Export Flow Credentials</template>
-        </FormRow>
-        <FormRow type="checkbox" v-if="parts.exportTemplate" v-model="parts.template">
+        <div :class="parts.flows?'opacity-100':'opacity-30'">
+            <FormRow type="checkbox" v-bind:disabled="!parts.flows" v-model="parts.credentials">
+                Credentials
+            </FormRow>
+        </div>
+        <FormRow type="checkbox" v-if="showTemplate" v-model="parts.template">
             Template
-            <template #description>Export Project Template</template>
         </FormRow>
-        <FormRow v-if="parts.showSecret" type="text" v-model="parts.credsSecret" v-bind:disabled="!parts.flows && !parts.creds">
+        <FormRow v-if="showSecret" type="text" v-model="parts.credsSecret" v-bind:disabled="!parts.flows && !parts.credentials">
             Secret
             <template #description>Provide a Secret to encrypt the exported Credentials</template>
         </FormRow>
-        <FormRow type="checkbox" v-model="parts.envVars">
-            Environment Variables
-            <template #description>Export Environment Variables</template>
-        </FormRow>
-        <FormRow class="ml-9" v-if="parts.envVars" type="radio" v-model="parts.envVarsKo" :value="false">
-            Keys and Values
+        <FormRow v-if="showSettings" type="checkbox" v-model="parts.settings">
```

Comment:
You've added "Project Settings" to this component, but there is no code in the api to do anything with `options.settings` that I can see

Reply:
Good point. Will fix that.